src/components/Table/GridTable.tsx

```diff
@@ -1018,6 +980,7 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
         maybeAddCardPadding(openCardStyles, "first", {
           ...maybeStickyHeaderStyles,
           ...(columnSizes ? Css.w(columnSizes[0]).$ : {}),
+          ...(openCardStyles.length === 0 ? getLeafCardStyles(leafCardStyles, "first") : {}),
```

Comment:
**question:** What does `openCardStyles` signify again in this instance? Is this `0` when it's a leaf node?